### PR TITLE
Fix the record preview

### LINKED ISSUE #4452: Mehrere Bearbeiten im Backend - Elemente löschen - Fehlermeldung
### Affected version(s)

4.13.2

### Description

Fehlermeldung im Backend bei Contao 4.13.0 und 4.13.2, PHP 7.4. Wenn man mehrere Elemente auswählt in der Elementansicht und diese dann löscht macht er bei der Auswahl von ca. über 8 Elementen eine Fehlermeldung und löscht dann auch nicht alle Elemente. 
Fehlermeldung:

```
[2022-04-03T17:34:00.839409+02:00] request.INFO: Matched route "contao_backend". {"route":"contao_backend","route_parameters":{"_route":"contao_backend","_scope":"backend","_token_check":true,"_controller":"Contao\\CoreBundle\\Controller\\BackendController::mainAction"},"request_uri":"http://…/contao?act=deleteAll&do=article&id=447&ref=G9NGodB3&rt=f31e.I6QCjxg9eO_pwGGQYFoIiWoNDRXHIjXG_uq5DbP30U8.eu8wu0kMTbeMoS-nEwU47zh8PXv1UWTzsIfcUoGymgJ80ne5aQUNuaz4Bg&table=tl_content","method":"GET"} []
[2022-04-03T17:34:00.845485+02:00] doctrine.INFO: Connecting with parameters array{"driver":"pdo_mysql","url":"pdo-mysql://…@localhost:3306/XXX","charset":"utf8mb4","host":"localhost","port":3306,"user":"XXX","password":"<redacted>","driverOptions":{"1002":"SET SESSION sql_mode=(SELECT CONCAT(@@sql_mode, ',TRADITIONAL'))","1013":false},"defaultTableOptions":{"charset":"utf8mb4","collate":"utf8mb4_unicode_ci","engine":"InnoDB","row_format":"DYNAMIC"},"dbname":"XXX"} {"params":{"driver":"pdo_mysql","url":"pdo-mysql://…@localhost:3306/XXX","charset":"utf8mb4","host":"localhost","port":3306,"user":"XXX","password":"<redacted>","driverOptions":{"1002":"SET SESSION sql_mode=(SELECT CONCAT(@@sql_mode, ',TRADITIONAL'))","1013":false},"defaultTableOptions":{"charset":"utf8mb4","collate":"utf8mb4_unicode_ci","engine":"InnoDB","row_format":"DYNAMIC"},"dbname":"XXX"}} []
[2022-04-03T17:34:00.897257+02:00] request.CRITICAL: Uncaught PHP Exception TypeError: "Argument 2 passed to Contao\CoreBundle\EventListener\DataContainer\RecordPreviewListener::compilePreview() must be of the type array, bool given, called in /www/htdocs/…/vendor/contao/core-bundle/src/EventListener/DataContainer/RecordPreviewListener.php on line 63" at /www/htdocs/…/vendor/contao/core-bundle/src/EventListener/DataContainer/RecordPreviewListener.php line 71 {"exception":"[object] (TypeError(code: 0): Argument 2 passed to Contao\\CoreBundle\\EventListener\\DataContainer\\RecordPreviewListener::compilePreview() must be of the type array, bool given, called in /www/htdocs/…/vendor/contao/core-bundle/src/EventListener/DataContainer/RecordPreviewListener.php on line 63 at /www/htdocs/…/vendor/contao/core-bundle/src/EventListener/DataContainer/RecordPreviewListener.php:71)"} []
```

Screenshot der Fehlermeldung anbei

### PR DESCRIPTION
**Pull request by bezin**
Fixes #4452

### CODE REVIEW

File: core-bundle/src/EventListener/DataContainer/RecordPreviewListener.php [hunk 1]
```diff
@@ -60,6 +60,10 @@ public function storePrecompiledRecordPreview(DataContainer $dc, string $undoId)
                 ->fetchAssociative()
             ;
 
+            if (!$row) {
+                return;
+            }
```
Comment (m-vo) [inline core-bundle/src/EventListener/DataContainer/RecordPreviewListener.php]: Should we move this and the database query above the `try` block?

Comment (bezin) [reply core-bundle/src/EventListener/DataContainer/RecordPreviewListener.php]: Seems possible yes. Other exceptions captured here might happen again when we try to update the database anyway.

Comment (bezin) [reply core-bundle/src/EventListener/DataContainer/RecordPreviewListener.php]: I updated accordingly

**Review by m-vo — APPROVED** [review]

**Comment by leofeyer** [comment]
Thank you @bezin.

**Pull request merged**